kahuna/public/js/crop/view.html

```diff
@@ -8,22 +8,22 @@
     <gr-top-bar-actions>
         <div class="top-bar-item side-padded">{{ ctrl.cropWidth() }} &times; {{ ctrl.cropHeight() }}</div>
 
-        <label class="top-bar-item clickable side-padded" title="landscape">
+        <label class="top-bar-item clickable side-padded titip-default titip-bottom" data-title="landscape [l]">
```

Comment:
I've seen shortcuts listed in both upper (soundcloud) and lower (twitter). Happy to convert to upper and get feedback from users.